libbeat/docs/loggingconfig.asciidoc

```diff
@@ -124,6 +125,11 @@ The number of most recent rotated log files to keep on disk. Older files are
 deleted during log rotation. The default value is 7. The `keepfiles` options has to be
 in the range of 2 to 1024 files.
 
+===== files.permissions
+
+The permission mask to apply when rotating log files. The default value is 0600. The 
+`permissions` option must be a valid file permissions mask.
```

Comment:
Can we add a link to a resource that describes what constitutes a valid file permissions mask? An example is always good, too.